API.md

```diff
@@ -0,0 +1,44 @@
+## Classes
```

Comment:
These docs are generated with jsdoc-to-markdown. I don't particularly love the output, but my goals was to not have to write them at all, but rather have them generated using inline comments (which I personally find to be more useful). Which is also the reason for a separate file ( then I didn't have to concatenate parts of a readme, though I should link to this in the readme.)